packages/currency/src/currencyManager.ts

```diff
@@ -172,6 +174,43 @@ export class CurrencyManager<TMeta = unknown> implements ICurrencyManager<TMeta>
     };
   }
 
+  /**
+   * Validates an address for a given currency.
+   * Throws if the currency is not given or if the currency is an ISO4217 currency.
+   * We allow undefined type for the "currency" argument to make it easier to chain with the
+   * CurrencyManager.from() method but if the argument is not defined it will throw an error.
+   */
+  static validateAddress(address: string, currency: CurrencyInput | undefined): boolean {
+    if (!currency) {
+      throw new Error('Could not validate an address of an undefined currency');
+    }
```

Comment:
maybe just don't accept undefined? I'm not  a huge fan of throwing in these cases, it kind of requires apps that use this helper to build a wrapper around this method

Reply:
done is https://github.com/RequestNetwork/requestNetwork/pull/608/commits/3b350dd8b9c3045c1fd4694f2a4367e7b572dd42 , please tell me if it's OK

Reply:
yep!